### PR TITLE
Some tweaks to the Obstacle Settings dialog

### PR DESCRIPTION
**Pull request by nyalldawson**
Supersedes https://github.com/qgis/QGIS/pull/36817

### CODE REVIEW

**Comment by DelazJ** [comment]
I was looking into it right now but I'm afraid I wouldn't have found this. Yet. Thanks

**Comment by nyalldawson** [comment]
> I was looking into it right now but I'm afraid I wouldn't have found this. Yet. Thanks

I have full confidence that you'll soon be doing these kind of changes in your sleep :laughing:

File: python/gui/auto_generated/labeling/qgslabelsettingswidgetbase.sip.in [hunk 1]
```diff
@@ -125,6 +125,14 @@ Constructor for QgsLabelSettingsWidgetDialog.
 :param parent: parent widget
 %End
 
+
+    QDialogButtonBox *buttonBox();
+%Docstring
+Return's the dialog's button box.
```
Comment (DelazJ) [inline python/gui/auto_generated/labeling/qgslabelsettingswidgetbase.sip.in:131]:
```suggestion
Returns the dialog's button box.
```

File: src/gui/labeling/qgslabelsettingswidgetbase.h [hunk 2]
```diff
@@ -152,6 +153,18 @@ class GUI_EXPORT QgsLabelSettingsWidgetDialog : public QDialog
      */
     QgsLabelSettingsWidgetDialog( QgsLabelSettingsWidgetBase *widget SIP_TRANSFER, QWidget *parent = nullptr );
 
+
+    /**
+     * Return's the dialog's button box.
```
Comment (DelazJ) [inline src/gui/labeling/qgslabelsettingswidgetbase.h:158]:
```suggestion
     * Returns the dialog's button box.
```

**Review by DelazJ — COMMENTED** [review]
Spotted a typo

File: src/gui/labeling/qgslabelsettingswidgetbase.h (hunk 2 above)
Comment (nyalldawson) [reply src/gui/labeling/qgslabelsettingswidgetbase.h:158]: Thanks!

**Pull request merged**